examples/post.v3.LR.historical.zppy_v3.cfg

```diff
@@ -0,0 +1,293 @@
+  climo_subsection = "atm_monthly_180x360_aave"
+  reference_data_path = '/lcrc/group/e3sm2/ac.zhang40/E3SMv3/v3.LR.historical_eu11rc13/post/atm/180x360_aave/clim'
+  # Group 2: climo atm diurnal
+  # "diurnal_cycle"
+  climo_diurnal_frequency = "diurnal_8xdaily"
+  climo_diurnal_subsection = "atm_monthly_diurnal_8xdaily_180x360_aave"
+  # Group 3: ts atm monthly
+  # "enso_diags" (not requested), "qbo", "area_mean_time_series" (not requested)
+  ts_num_years = 5
+  ts_num_years_ref = 30
+  ts_subsection = "atm_monthly_180x360_aave"
+  # Group 4: ts atm daily
+  # "tropical_subseasonal" (not requested)
+  # Group 5: ts rof monthly
+  # "streamflow" (not requested)
+  # Group 6: tc_analysis
+  # "tc_analysis" (not requested)
+  # Other parameters:
+  diff_title = 'Difference'
+  grid = '180x360_aave'
+  ref_final_yr = 2014
+  ref_name = 'v3.LR.historical_0051'
+  ref_start_yr = 1985
+  ref_years = "1985-2014",
+  run_type = "model_vs_model"
+  short_name = 'v3.LR.historical'
+  short_ref_name = 'v3.LR.historical_0051_ref'
+  tag = 'model_vs_model'
+
+  [[ lnd_monthly_mvm_lnd ]]
+  # Depends on the climo task's land_monthly_climo subtask.
+  sets = "lat_lon_land",
+  climo_subsection = "land_monthly_climo"
+  # Other parameters:
+  diff_title = "Difference"
+  grid = 'native'
+  reference_data_path = "/lcrc/group/e3sm/ac.zhang40/tests/20231209.v3.LR.piControl-spinup.chrysalis_land_diags/post/lnd/native/clim"
+  ref_final_yr = 0100
+  ref_name = "20231209.v3.LR.piControl-spinup.chrysalis"
+  ref_start_yr = 0051
+  ref_years = "0051-0100",
+  run_type = "model_vs_model"
+  short_ref_name = "20231209.v3.LR.piControl-spinup"
+  short_name = v3.LR.piControl
+  swap_test_ref = False
+  tag = "model_vs_model"
+
+[mpas_analysis]
+active = True
+anomalyRefYear = 1985
+climo_years = "1985-2014",
+enso_years = "1985-2014",
+mesh = "IcoswISC30E3r5"
+parallelTaskCount = 6
+shortTermArchive = True
+ts_years = "1985-2014",
+walltime = "4:00:00"
+
+[global_time_series]
+# This task depends on the mpas_analysis task and the ts task's atm_monthly_glb and lnd_monthly_glb subtasks.
+# 2 DEPRECATED PARAMETERS in zppy v3.0.0 :
+# atmosphere_only (now just remove the ocean plots when setting plots_original)
+# plot_names (plot names should now be explicitly set via the plots_atm/ice/lnd/ocn parameters)
+active = True
+climo_years ="1985-2014",
+environment_commands = "source /gpfs/fs1/home/ac.forsyth2/miniforge3/etc/profile.d/conda.sh; conda activate zi_plots_lnd_20250326"
+experiment_name = "v3.LR.historical_0051"
+figstr = "v3.LR.historical_0051"
+make_viewer = True
+moc_file = "mocTimeSeries_1985-2014.nc"
+plots_atm = "TREFHT" # This will plot in the atm component; it has no effect on the original plots
+plots_lnd = "FSH,RH2M,LAISHA,LAISUN,QINTR,QOVER,QRUNOFF,QSOIL,QVEGE,QVEGT,SOILWATER_10CM,TSA,H2OSNO,TOTLITC,CWDC,SOIL1C,SOIL2C,SOIL3C,SOIL4C,WOOD_HARVESTC,TOTVEGC,NBP,GPP,AR,HR" # Set this to "all" to plot all land variables
```

Comment:
@chengzhuzhang I changed this from "all" -- I think given the performance issues and this just being an example cfg, we can reduce the number of variables here. I can change the walltime too.